package.json

```diff
@@ -16,7 +17,7 @@
     "lint:css": "stylelint '_sass/**/*.scss' 'assets/scss/**/*.scss'",
     "typecheck": "tsc",
     "pretest": "if [ -z $SKIP_BUILD ]; then make build; fi",
-    "test": "BABEL_ENV=test jest"
+    "test": "NODE_ENV=test node --import=esbuild-esm-loader/register --import=./spec/spec_helper.js --test spec/**/*.{js,ts,tsx}"
```

Comment:
should we add in `.jsx` for completeness or would we want to leave it out to discourage tests written in that file format?

Reply:
I think I lean toward the latter, that omitting has the side-benefit of discouraging and forcing a move toward TypeScript.